data-prepper-core/src/integrationTest/java/com/amazon/dataprepper/integration/EndToEndRawSpanTest.java

```diff
@@ -152,8 +154,10 @@ private void refreshIndices(final RestHighLevelClient restHighLevelClient) throw
     }
 
     private void sendExportTraceServiceRequestToSource(final int port, final ExportTraceServiceRequest request) {
-        Clients.newClient(String.format("gproto+http://127.0.0.1:%d/", port),
-                TraceServiceGrpc.TraceServiceBlockingStub.class).export(request);
+        TraceServiceGrpc.TraceServiceBlockingStub client = Clients.builder(String.format("gproto+http://127.0.0.1:%d/", port))
+                .decorator(RetryingClient.newDecorator(RetryRule.failsafe()))
```

Comment:
I like this change overall!

The documentation on this wasn't clear regarding how many times it tries. Is this limited?
If we are unsure as to the number of retries, we should be sure that the test itself has a [JUnit @Timeout](https://junit.org/junit5/docs/5.5.1/api/org/junit/jupiter/api/Timeout.html).

Reply:
Yes. Here is the specification: https://javadoc.io/static/com.linecorp.armeria/armeria-javadoc/1.11.0/com/linecorp/armeria/common/Flags.html#defaultBackoffSpec()